ambari-server/src/main/java/org/apache/ambari/server/api/services/RootClusterSettingService.java

```diff
@@ -70,7 +70,7 @@ public RootClusterSettingService() {
           response = ReadOnlyConfigurationResponse.ReadOnlyConfigurationResponseSwagger.class, responseContainer = RESPONSE_CONTAINER_LIST)
   @ApiImplicitParams({
           @ApiImplicitParam(name = QUERY_FIELDS, value = QUERY_FILTER_DESCRIPTION, dataType = DATA_TYPE_STRING,
-                  paramType = PARAM_TYPE_QUERY, defaultValue = MpackResourceProvider.MPACK_ID),
+                  paramType = PARAM_TYPE_QUERY, defaultValue = MpackResourceProvider.MPACK_RESOURCE_ID),
```

Comment:
I think this one was an error. Mpackid will not come in the context of clustersettings.

Reply:
@mradha25 you are right. I just did a find an replace. @swapanshridhar can you check this and fix it in your next patch?

Reply:
@jayush Sure. Vl take care.